packages/init/src/steps/loginOrBypass.ts

```diff
@@ -1,5 +1,6 @@
-import { Utils, Auth, Models } from "@slicemachine/core";
+import { Models } from "@slicemachine/core";
 import { validateSessionAndGetProfile } from "../utils/communication";
+import { logs, Auth } from "../utils";
```

Comment:
amazing what you did there, pulling out this whole part from the Core is awesome !!